src/components/Pagination/Pagination.tsx

```diff
@@ -35,6 +36,8 @@ export interface PaginationDescriptor {
   onNext?(): void;
   /** Callback when previous button is clicked */
   onPrevious?(): void;
+  /** Text to provide more context in between the arrow buttons */
+  label?: string;
```

Comment:
Is this the best prop name for this, it's not a `label` in the form sense of the word, maybe `paginationText` or `currentPageText`?

These aren't the best suggestions, and I'm ok with `label` lacking a better name, I just wonder if this is the _best_ name 😅

Reply:
I agree, label could be confusing. How about something like `indicatorText`?

Reply:
I would agree with `paginationText`, `indicatorText` is too vague.

Reply:
^ We chatted about this on slack and decided to stick with label